pkg/kubelet/rkt/rkt.go

```diff
@@ -789,7 +789,12 @@ func (r *runtime) PullImage(image kubecontainer.ImageSpec, pullSecrets []api.Sec
 		tag = "latest"
 	}
 
-	creds, ok := r.dockerKeyring.Lookup(repoToPull)
+	keyring, err := credentialprovider.MakeDockerKeyring(pullSecrets, r.dockerKeyring)
```

Comment:
@lavalamp @pmorie @deads2k This is because rkt can pull and run docker images with docker credentials, I read the keyring here and store it in the rkt's auth config files for docker (https://github.com/coreos/rkt/blob/master/Documentation/configuration.md#rktkind-dockerauth)